src/Workspaces/CSharpTest/Formatting/FormattingTests.cs

```diff
@@ -9973,5 +9973,61 @@ public async Task RecordStruct()
 record  struct  R(int X);
 ");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.Formatting)]
+        public async Task LambdaReturnType_01()
+        {
+            await AssertFormatAsync(
+@"class Program
+{
+    Delegate D = void () => { };
+}",
+@"class Program
+{
+    Delegate D = void  ()  =>  {  };
```

Comment:
wacky :)

Reply:
(oh wait, is this one of those weir dtest harnesses that puts the 'expected' first?

Reply:
Yes, `expected` is the first parameter.